CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
 ### Unreleased
   - Add codacy badge
```

Comment:
Not yours, but may be worth removing this line - it tends to imply an amendment on something that hasn't been released